examples/sign/ed25519.eg.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * @title Ed25519 Signatures
+ * @stability nearing
+ *
+ * Utilize an Ed25519 library for signing.
+ */
+
+import { assert } from "asserts"
+import { Rune } from "capi"
+import { signature } from "capi/patterns/signature/polkadot.ts"
+import * as ed from "npm:@noble/ed25519"
+import { sha512 } from "npm:@noble/hashes/sha512"
+import { Balances, createUsers, System, types } from "westend_dev/mod.js"
+
+const { alexa, billy } = await createUsers()
+
+const billyFree = System.Account
+  .value(billy.publicKey)
+  .unhandle(undefined)
+  .access("data", "free")
+
+const billyFreeInitial = await billyFree.run()
+
+// Bring your own Ed25519 implementation. In this case, we utilize
+// [`noble-ed25519`](https://github.com/paulmillr/noble-ed25519), which
+// requires us to specify a sync sha512 hasher implementation.
+ed.etc.sha512Sync = (...msgs) => sha512(ed.etc.concatBytes(...msgs))
```

Comment:
This wasn't necessary before?

Reply:
Correct, but this enables sync use of the ed25519 lib, which I believe is preferable. Also, it will highlight that this is mandatorily BYO.